presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -128,6 +128,7 @@
     public static final String REQUIRED_WORKERS_COUNT = "required_workers_count";
     public static final String REQUIRED_WORKERS_MAX_WAIT_TIME = "required_workers_max_wait_time";
     public static final String COST_ESTIMATION_WORKER_COUNT = "cost_estimation_worker_count";
+    public static final String ENABLE_LEGACY_TIMESTAMP_FORMAT_SUPPORT = "enable_legacy_timestamp_support";
```

Comment:
`legacy` already has a special meaning for timestamps. It's used to indicate timestamp semantics where timestamps are treated like instants.

I would suggest a different name, such as "omit_timestamp_precision"

Reply:
done.